### PR TITLE
Feat: create install and uninstall app

### PR DESCRIPTION
**Pull request by didd**
## Description

- Create install and uninstall app

## Checklist

- [X] I have read the **README** document
- [X] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [X] All new and existing tests passed
- [X] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/design-system-components/src/components/InstallApp/install-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      />
```
Comment (kenshyx) [inline ui/design-system-components/src/components/InstallApp/install-icon.tsx]: can we avoid base64 encoded images? :/

Comment (kenshyx) [reply ui/design-system-components/src/components/InstallApp/install-icon.tsx]: also https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/xlink:href is deprecated

Comment (didd) [reply ui/design-system-components/src/components/InstallApp/install-icon.tsx]: Agreed. I will ask Basma for a WebP version. I took the svg directly from Figma.

Comment (kenshyx) [reply ui/design-system-components/src/components/InstallApp/install-icon.tsx]: webp might not work with svgImage

Comment (didd) [reply ui/design-system-components/src/components/InstallApp/install-icon.tsx]: I had a chat with Basma and she explained those are actually lottie file animation. Never heard about this before :) And this lottie files are available elsewhere in the apps as well. I found out that there is a [react component](https://github.com/LottieFiles/lottie-react) for such files. We will need to discuss the implementation detail and in the mean time I have replaced the svg's with a placeholder.

Comment (didd) [reply ui/design-system-components/src/components/InstallApp/install-icon.tsx]: Finally, I have converted all the icons into WebP and used `img` tag to display them. The animations are complex and creating them in css will not work.

**Review by kenshyx — APPROVED** [review]

**Review by kenshyx — APPROVED** [review]

**Pull request merged**